ggshield/verticals/hmsl/output.py

```diff
@@ -23,6 +23,18 @@
     URL: "{url}"
 """
 
+TOO_MANY_SECRET_THRESHOLD = 100
```

Comment:
nit: should be `TOO_MANY_SECRETS_THRESHOLD`. Same for `TOO_MANY_SECRET_TEMPLATE`.